pages/news/index.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <div>
+    <v-container>
+      <NewsFilter :reloadNews="getNews" />
+      <v-row v-if="isNewsLoading">
+        <v-col v-for="index in 2" :key="index" class="col-12 col-md-6">
+          <NewsCardSkeleton />
+        </v-col>
+      </v-row>
+      <v-row v-else>
+        <v-col
+          v-for="(news, index) in newsData"
+          :key="`${news?.source.id}-${index}`"
+        >
+          <NewsCard :news="news" />
+        </v-col>
+      </v-row>
+    </v-container>
+  </div>
+</template>
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator'
+// import Vue from 'vue'
+import { mapGetters, mapMutations } from 'vuex'
+
+import { NewsInterface } from '~/interfaces'
+// import newDataFromFile from '~/static/news'
+import NewsCardSkeleton from '~/components/skeleton/NewsCardSkeleton.vue'
+
+@Component({
+  components: { NewsCardSkeleton },
+  computed: {
+    ...mapGetters(['isNewsLoading', 'selectedCategory', 'selectedCountry']),
+  },
+  methods: {
+    ...mapMutations(['updateIsnewsLoading']),
+  },
+})
+export default class extends Vue {
+  newsData: Array<NewsInterface | null> = []
+
+  isNewsLoading!: boolean
+  selectedCategory!: string
+  selectedCountry!: string
+  updateIsnewsLoading!: Function
+
+  async getNews() {
+    const newsData = await this.$axios.$get(
```

Comment:
can we pass a limit to API, that returns data up to limit

Reply:
yes, now added to get max 100 data

Reply:
resolved